Submodels/MTFLO_call.py

```
@@ -1,20 +1,183 @@
 
 import subprocess
 import os
 
 class MTFLO_call():
-    pass
\ No newline at end of file
+    """
+    Class to handle the interface between Python and the MTFLO executable.
+    """
+
+    def __init__(self, *args,
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)